lib/fake_braspag/sales.rb

```diff
@@ -40,6 +40,16 @@ class Sales < Sinatra::Base
       end
     end
 
+    put '/:PaymentId/conciliate' do
+      order = Order.find(params[:PaymentId])
+      if order.present? && order.PaymentMethod == 'Boleto' && ResponseToggler.enabled?('conciliate')
```

Comment:
And if a 404 error will not be handled, we could even move the ` Order.find(params[:PaymentId])` to `if` block
```ruby
if ResponseToggler.enabled?('conciliate') && order = Order.find(params[:PaymentId]) && order.PaymentMethod == 'Boleto'
  ...
end
```